sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -102,7 +102,20 @@ def generate_prompts(
     return all_prompts, all_negative_prompts
 
 
+already_loaded = False
+
+
 class Script(scripts.Script):
+    def __init__(self):
+        global already_loaded
+        if not already_loaded:
+            already_loaded = True
```

Comment:
I suppose this would happen if Reload UI is hit? Maybe a comment to that effect..?

Reply:
This is something weird that I can't figure out. My class gets created twice, not sure it's my setup specifically. I haven't investigated in detail

Reply:
You could slap a `traceback.print_stack()` in `__init__` to see why/when :)